### PR TITLE
Addition of flipr hub with switch and select platform

### PR DESCRIPTION
**Pull request by cnico**
## Proposed change

Addition of flipr hub documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125866 and https://github.com/home-assistant/core/pull/126237
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced documentation for the Flipr integration, clarifying device functionalities and data publishing methods.
	- Added details on controlling pool equipment and monitoring parameters like pH and chlorine levels.
	- Clarified prerequisites for integration usage and reorganized content for better usability.
	- Introduced new sections on the Flipr Hub's functionalities and actions for refreshing cloud data.

### CODE REVIEW

File: source/_integrations/flipr.markdown [hunk 1]
```diff
@@ -30,14 +50,16 @@ There is currently support for the following information within Home Assistant:
...
 
-- [Google](https://play.google.com/store/apps/details?id=com.goflipr.flipr)
-- [Apple](https://apps.apple.com/fr/app/flipr/id1225898851)
+- **Turn on/off** the switch inside the Hub and automatically set the Hub in manual mode
+- **Choose** the mode of the Hub between auto, planning and manual.
 
-{% include integrations/config_flow.md %}
+## Actions
+
+The polling of the cloud data is configured to check every 15 minutes. If you want to force a refresh of the data, you can use the `homeassistant.update_entity` action.
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
The cloud data is polled every 15 minutes. If you want to force a refresh of the data, you can use the `homeassistant.update_entity` action.
```

File: source/_integrations/flipr.markdown [hunk 2]
```diff
@@ -11,15 +11,35 @@ ha_codeowners:
...
+
+{% include integrations/config_flow.md %}
+
+## Flipr
+
 Flipr sends data like pH, chlorine or temperature to a cloud server on a regular basis in order to monitor your pool.
-This integration gives you access to this information on Home Assistant in addition to the vendor's smartphone application.
 
-There is currently support for the following information within Home Assistant:
+There is currently support for the following information within Home Assistant via **sensors** and **binary_sensors** :
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown:42]: Could you add a section called ## Entities, and below that a separate section for each type, ### Sensors, ### Binary sensors, etc and list them with name and description?

Comment (cnico) [reply source/_integrations/flipr.markdown:42]: Pushed a correction for it. Is it ok for you ?

Comment (c0ffeeca7) [reply source/_integrations/flipr.markdown:42]: yes, thank you! Could you also give the units for the chlorine level?

File: source/_integrations/flipr.markdown [hunk 3]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+
+There is currently support for the following information within Home Assistant via **sensors** and **binary_sensors** :
+
+## Entities
+
+### Sensors
+
+- Chlorine : the chlorine level
+- pH : the pH level of the water
+- Water Temperature : the temperature of the water
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- Water temperature : the temperature of the water
```
we use (sentence-style capitalization)[https://learn.microsoft.com/en-us/style-guide/capitalization]

File: source/_integrations/flipr.markdown [hunk 4]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+- Red OX : the redox level of the water in mV
+- Last measure date : the date of the last measure taken by the flipr
+- Battery Level : the battery level of the flipr in percentage
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- Battery level : the battery level of the flipr in percentage
```

File: source/_integrations/flipr.markdown [hunk 5]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+
+### Binary sensors
+
+- pH Status : an indicator if pH level is normal, too loo or too high
+- Chlorine Status : an indicator if chlorine level is normal, too loo or too high
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- Chlorine status : an indicator if the chlorine level is normal, too low, or too high
```

File: source/_integrations/flipr.markdown [hunk 6]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+
+## Hub
+
+The Hub lets you handle your equipment (pump, heater, light, etc.) from Home Assistant and all of the automation you can imagine.
+
+- **Turn on/off** the switch inside the Hub and automatically set the Hub in manual mode
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- **Turn on/off** the switch inside the Hub and automatically set the Hub in manual mode.
```

File: source/_integrations/flipr.markdown [hunk 7]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+## Entities
+
+### Sensors
+
+- Chlorine : the chlorine level
+- pH : the pH level of the water
+- Water temperature : the temperature of the water
+- Red OX : the redox level of the water in mV
+- Last measure date : the date of the last measure taken by the flipr
+- Battery level : the battery level of the flipr in percentage
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- **Chlorine**: the chlorine level
- **pH**: the pH level of the water
- **Water temperature**: the temperature of the water
- **Red OX**: the redox level of the water in mV
- **Last measure date**: the date of the last measure taken by the flipr
- **Battery level**: the battery level of the flipr in percentage
```

File: source/_integrations/flipr.markdown [hunk 8]
```diff
@@ -39,6 +35,39 @@ You will need to use the standalone app for this device to register a username a
...
+
+### Binary sensors
+
+- pH Status : an indicator if pH level is normal, too loo or too high
+- Chlorine status : an indicator if the chlorine level is normal, too low, or too high
```
Comment (c0ffeeca7) [inline source/_integrations/flipr.markdown]:
```suggestion
- **pH Status**: an indicator if the pH level is normal, too low, or too high
- **Chlorine status**: an indicator if the chlorine level is normal, too low, or too high
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @cnico :+1:

**Pull request merged**